controllers/cinder_controller.go

```diff
@@ -278,7 +280,8 @@ func (r *CinderReconciler) SetupWithManager(mgr ctrl.Manager) error {
 }
 
 func (r *CinderReconciler) reconcileDelete(ctx context.Context, instance *cinderv1beta1.Cinder, helper *helper.Helper) (ctrl.Result, error) {
-	r.Log.Info(fmt.Sprintf("Reconciling Service '%s' delete", instance.Name))
+	log := r.getLog(ctx)
```

Comment:
Why not use the logger that's available in the helper?

```
    helper.Log.Info(...)
```

Reply:
For me the Helper is a dumping ground of unrelated utility functions. See https://wiki.c2.com/?AbuseOfUtilityClasses So my ultimate goal is not to add things to it but actually dissolve it in the long run.

Reply:
I had a stab at having a separate logger package https://github.com/openstack-k8s-operators/lib-common/pull/125  but it turned out that we already get a proper logger via the ctx anyway so that package would be fairly empty.

Reply:
So where does that leave us? It seems reasonable for every controller to have its own logger function, and the question is how to provide easy access to that function. I agree the Helper shouldn't be a dumping ground, but if we avoid using it then we'll need some other way to forward.

Reply:
the `ctx` is already passed around and the `ctx` already contains a a logger to use. So the `r.getLog(ctx)` is good for me.

Reply:
I.e for me the difference between ` helper.Log.Info(...)` an `r.getLog(ctx).Info()` is that ctx is the framework's way to pass a logger to us and we already have the context in most places, so pulling the logger out of context and adding it to the helper seems like an unnecessary complication to me.